webdriver-extension/index.bs

```diff
@@ -0,0 +1,546 @@
+      dictionary AbsoluteOrientationReadingValues {
+        required FrozenArray&lt;double>? quaternion;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Relative Orientation Sensor ### {#mock-relative-orientation-sensor}
+The <dfn for="MockSensorType" enum-value>"relative-orientation"</dfn> type is the mock sensor type
+associated with the usage of the <a>RelativeOrientationSensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Values</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary RelativeOrientationReadingValues : AbsoluteOrientationReadingValues {
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Geolocation Sensor ### {#mock-geolocation-sensor}
+The <dfn for="MockSensorType" enum-value>"geolocation"</dfn> type is the mock sensor type
+associated with the usage of the <a>GeolocationSensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Values</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary GeolocationReadingValues {
+        required double? latitude;
+        required double? longitude;
+        required double? altitude;
+        required double? accuracy;
+        required double? altitudeAccuracy;
+        required double? heading;
+        required double? speed;
+      };
+    </pre>
+  </dd>
+</dl>
+
+### Proximity Sensor ### {#mock-proximity-sensor}
+The <dfn for="MockSensorType" enum-value>"proximity"</dfn> type is the mock sensor type
+associated with the usage of the <a>ProximitySensor</a> interface.
+<dl>
+  <dt><a>Mock Sensor Reading Values</a></dt>
+  <dd>
+    <pre class="idl">
+      dictionary ProximityReadingValues {
+        required double? distance;
+        required double? max;
+        required boolean? near;
+      };
+    </pre>
+  </dd>
+</dl>
+
+Extensions {#webdriver-extensions}
+=====================
+
+## Commands ## {#mock-sensor-commands}
+
+### Create mock sensor ### {#create-mock-sensor-command}
+
+<table>
+  <tbody>
+    <tr>
+      <th>HTTP Method</th>
+      <th><a lt="extension command URI template">URI Template</a></th>
+    </tr>
+    <tr>
+      <td>POST</td>
+      <td>/session/{session id}/sensor</td>
+    </tr>
+  </tbody>
+</table>
+
+The <dfn>create mock sensor</dfn> <a>extension command</a> simulates user
+creation of a <a>mock sensor</a>.
+
+The <a>remote end steps</a> are:
+1. Let |configuration| be the |configuration| argument, <a>converted to an IDL value</a>
+   of type {{MockSensorConfiguration}}. If this throws an exception, return a
+   <a>WebDriver error</a> with <a>WebDriver error code</a> <a>invalid argument</a>.
+2. Let |type| be the |configuration|.{{MockSensorConfiguration/mockSensorType}}, if the <a>current browsing context</a>
+   already has this |type| of [=mock sensor=], return a <a>WebDriver error</a> with <a>WebDriver error code</a>
+   [=mock sensor already created=].
+3. Run these sub-steps [=in parallel=] to create a [=mock sensor=] in <a>current browsing context</a>:
+  1. Set the [=mock sensor type|mock sensor's type=] to |type|.
```

Comment:
This seems like a similar pattern as the "Sensor object" reference below. I would expect a step that explicitly created an instance of `[=mock sensor=]` and stored it in a variable like `<var>mock</var>`